web/app.yaml

```diff
@@ -1,5 +1,3 @@
-application: dart-pad
```

Comment:
These are no longer used? Some appengine change?

Reply:
Yep, it has been updated: https://cloud.google.com/appengine/docs/standard/python/config/appref